plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -157,4 +158,21 @@ private static void executeBigQuerySql(BigQuery bigquery, String query)
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public void testShowCreateTable()
+    {
+        assertThat((String) computeActual("SHOW CREATE TABLE orders").getOnlyValue())
+                .matches("CREATE TABLE \\w+\\.\\w+\\.orders \\Q(\n" +
```

Comment:
please use `.isEqualTo` -- no need for pattern-based matching when we know what the output is going to be